test/integration/agent/route_test.go

```diff
@@ -440,33 +407,23 @@ func TestRouteTablePolicyOnly(t *testing.T) {
 
 	cLink := &netlink.Dummy{}
 	cLink.Name = "containerLink"
-	err = netlink.LinkAdd(cLink)
-	if err == nil {
-		err = netlink.LinkSetUp(cLink)
-	}
-	if err != nil {
-		t.Error(err)
-	}
+	assert.NoError(t, netlink.LinkAdd(cLink), "error when creating link")
```

Comment:
Maybe we should keep the same error message format like others?
https://github.com/vmware-tanzu/antrea/pull/1726/files#diff-fe40f0ca64908795bfc89d1e43e60f9370b74799be1f574d9d51ba9851844d8fR367
`"link creation failed"`